File: LeetCode/0344. Reverse String/JavaScript/02. Reverse String.js
var reverseString = function(s) {
    let startArray = 0;
    let endedArray = s.length - 1;
    while(startArray < endedArray){
        let tmp = s[startArray];
        s[startArray] = s[endedArray];
        s[endedArray] = tmp;
        startArray++;
        endedArray--;
    }
};

File: LeetCode/0344. Reverse String/C#/02. Reverse String.cs
void ReverseString(char[] s)
{
    int start = 0, end = s.Length - 1;
    while (start < end)
    {
        char tmp = s[start];
        s[start] = s[end];
        s[end] = tmp;
        start++;
        end--;
    }
}